src/shared/components/challenge-detail/Checkpoints/index.jsx

```diff
@@ -1,19 +1,15 @@
 import React from 'react';
 import PT from 'prop-types';
 
-import jquery from 'jquery';
```

Comment:
Ok, but what about removing `jquery` from `package.json`, `package-lock.json` and `.eslintrc.json`? I believe, it is not used anywhere else in our app, and it is not supposed to be. Do you see anything wrong with it?

Reply:
It is in fact used somewhere else, i.e. not directly but by a package react-slick. I'm removing it from package.json, but it will remain in package-lock.json.

Reply:
Ha! `package-lock.json` mentions separately package versions installed as direct dependencies, and those used as dependencies of dependencies, so it is actually straightforward to just remove what we want to remove. Though, don't worry, I'll do this last small fix myself, and if everything works fine, I'll merge the PR.